ttnn/cpp/ttnn/decorators.hpp

```diff
@@ -113,6 +113,16 @@ auto map_execute_on_worker_thread_return_to_launch_op_return(const T&& value) ->
         return value;
     } else if constexpr (std::is_same_v<std::decay_t<decltype(value)>, Tensor>) {
         return {value};
+    } else if constexpr (std::is_same_v<std::decay_t<decltype(value)>, OptionalTensors>) {
+        Tensors output_tensors;
+        auto size = value.size();
+        output_tensors.reserve(size);
+
+        auto dummy_tensor = Tensor();
```

Comment:
Yeah, this part does not look good to me. 

@dmakoviichuk-tt i remember you added a support for a return of vector of optional tensors ~2 months ago. Can you please take a look? This blocks Moreh and id appreciate your feedback here.

Reply:
I added support for the launch_op and for a few more cases.
Seems like decorators changed a lot since this time.